lib/tapioca/commands/todo.rb

```diff
@@ -74,10 +74,10 @@ def unresolved_constants
           .each_line
           .map do |line|
             next if line.include?("<")
-            next if line.include?("class_of")
-            line.strip.gsub("T.untyped::", "")
+            line.strip.gsub("T.untyped::", "").gsub(/T\.class_of\(([:\w]+)\)/, '\1')
```

Comment:
Nitpick: Can we have each `gsub` on its own line and add a comment for each line about what the processing is that they are doing?

Personally, I find that I need to really think about regexps to understand what they are doing and I would have still not understood why they are needed in the first place.

Reply:
Will do 👍

Do you know what the gsub for `T.untyped::` is doing? It was there before and after taking it out the whole suite still passes so I'm not sure what it's doing. Could've been something that has since changed in sorbet 🤔 cc @Morriar since I think you had initially added it.

Reply:
Yeah, maybe @Morriar would remember but I have no idea why we are stripping `T.untyped::` tbh.

Thanks for making the change.